sunpy/timeseries/tests/test_timeseriesmetadata.py

```diff
@@ -158,12 +158,30 @@ def test_append_invalid_timerange(basic_1_md):
 # Test TimeSeriesMetaData Concatenation
 # =============================================================================
 
-def test_concatenate(basic_ascending_append_md, basic_4_md, complex_append_md):
+def test_concatenate_ts(basic_ascending_append_md, basic_4_md, complex_append_md):
     concatenated = copy.deepcopy(basic_ascending_append_md)
     concatenated = concatenated.concatenate(basic_4_md)
     assert concatenated == complex_append_md
 
 
+def test_concatenate_list(basic_ascending_append_md, basic_3_md, basic_4_md, complex_append_md):
+    concatenated = copy.deepcopy(basic_ascending_append_md)
+    concatenated = concatenated.concatenate([basic_3_md, basic_4_md])
+    assert concatenated == complex_append_md
+
+
+def test_concatenate_invalid_type(basic_ascending_append_md):
+    concatenated = copy.deepcopy(basic_ascending_append_md)
+    with pytest.raises(TypeError):
```

Comment:
It is worth to match the message as well just to make sure you are raising the correct error.